package.json

```diff
@@ -17,7 +18,9 @@
     "package": "cross-env NODE_ENV=production node -r babel-register -r babel-polyfill webpack/package.js",
     "package-all": "npm run package -- --all",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
+    "preinstall": "which electron || npm install electron@1.4.12",
```

Comment:
Ok, isn't the point of this PR to avoid having to support multiple ways to run Daedalus? In the PR description you're saying: "We now require to use Nix for building the bridge, to be sure the build is reproducible", so what is the scenario of not needing Nix for development?